fe/src/main/java/org/apache/doris/system/Backend.java

```diff
@@ -346,6 +347,23 @@ public double getMaxDiskUsedPct() {
         return maxPct;
     }
 
+    public boolean checkDiskExceedLimitByStorageMedium(TStorageMedium storageMedium) {
```

Comment:
Add comment to explain the return value of this method.
the name `checkDiskExceedLimitByStorageMedium` is a little bit confused.

Reply:
done